ui/round/src/view/main.ts

```diff
@@ -57,3 +58,11 @@ export function main(ctrl: RoundController): VNode {
         ctrl.keyboardMove && renderKeyboardMove(ctrl.keyboardMove),
       ]);
 }
+
+export function endGameView() {
+  $('div.round__app__board.main-board.blindfold').removeClass('blindfold');
```

Comment:
why are we hacking into DOM that should be managed by snabbdom?

Reply:
No good reason. Rather than ensure the blindfold flag is stripped serverside on game completion, I took the easy way.

Reply:
`$('body').toggleClass('zen')` is a necessary dom hack because `body` is outside the round dom that snabbdom manages.

But `'div.round__app__board.main-board' + (hideBoard ? '.blindfold' : '')` is managed by snabbdom, so hacking it is dangerous and fortunately not needed.
Based on that current code:
```ts
  const hideBoard = ctrl.data.player.blindfold && playable(ctrl.data);
  [...]
        h('div.round__app__board.main-board' + (hideBoard ? '.blindfold' : ''),
```
it looks like snabbdom already takes care of adding and removing the `blindfold` class, based on a very reasonable `hideBoard` boolean check.
So... it should be enough? isn't it?

Reply:
Nor did I think to change the boolean locally and redraw, it's just shitty code.